smarctlide-monitoring/src/main/java/eu/smartclide/contexthandling/ServiceMain.java

```diff
@@ -0,0 +1,132 @@
+public class ServiceMain {
+    private static final Logger logger = LoggerFactory.getLogger(ServiceMain.class);
+    private static Server server;
+    private static IAmIMonitoringService service;
+    private static AmIMonitoringDataRepositoryServiceWrapper monitoringDataRepository;
+    private static IAmIMonitoringDataRepositoryService<IMonitoringDataModel<?, ?>> reposService;
+
+    private static void initialize() {
+        Properties props = System.getProperties();
+        props.setProperty("org.apache.cxf.stax.allowInsecureParser", "true");
+
+        org.apache.log4j.BasicConfigurator.configure();
+
+        String smartclideConfigPath = "resources";
+
+        // Environment Variable
+        String smartclideHome = System.getenv("SMARTCLIDE_HOME");
+        if (smartclideHome != null && Files.exists(Paths.get(smartclideHome))) {
+            smartclideConfigPath = smartclideHome + File.separator + "config";
+        // Linux config directory /var/lib/smartclide
+        } else if (Files.exists(Paths.get("/var/lib/smartclide"))) {
+            smartclideHome = "/var/lib/smartclide";
+            smartclideConfigPath = smartclideHome + File.separator + "config";
+        // Linux config directory /opt/smartclide/config
+        } else if (Files.exists(Paths.get("/opt/smartclide"))) {
+            smartclideHome = "/opt/smartclide";
+            smartclideConfigPath = smartclideHome + File.separator + "config";
+        // Windows COnfigu Directories
+        } else if (Files.exists(Paths.get("C:\\ProgramData\\smartclide"))) {
+            smartclideHome = "C:\\ProgramData\\smartclide";
+            smartclideConfigPath = smartclideHome + File.separator + "config";
+        }
+
+        AmIMonitoringConfiguration amionfig = new AmIMonitoringConfiguration();
+        amionfig.setId("SMARTCLIDE");
+        amionfig.setServiceConfiguration(readFile(smartclideConfigPath + File.separator + "monitoring-config.xml"));
+
+        File configFile = new File(
+                smartclideConfigPath + File.separator + "services-config.xml");
+        String filepath = configFile.getPath();
+        SWServiceContainer serviceContainer = new SWServiceContainer(
+                "AmI-repository", filepath);
+        ServiceManager.getLSWServiceContainer().add(serviceContainer);
+        server = ServiceManager.registerWebservice(serviceContainer);
+
+
+        for (SWServiceContainer container : ServiceManager.getLSWServiceContainer()) {
+            if (Objects.requireNonNull(container.getServerClass()).toString()
+                    .contains("AmIMonitoringDataRepository")) {
+                reposService = ServiceManager.getWebservice(container);
+            }
+        }
+
+        server = ServiceManager.registerWebservice(AmIMonitoringService.class);
+        service = ServiceManager.getWebservice(IAmIMonitoringService.class);
+        service.configureService(amionfig);
+    }
+
+    private static String readFile(String filename) {
```

Comment:
this could be simplified using `Files.readString(Path.of(filename), StandardCharsets.UTF_8)`

Reply:
done

Reply:
I don't see any change. Did you maybe forget to push a commit?

Reply:
changes were pushed already